site/nav.yml

```diff
@@ -32,18 +32,18 @@ nav:
   - Talks: talks.md
   - Vendors: vendors.md
   - Project: 
-    - Join: community.md
+    - Community: community.md
     - Spec: spec.md
     - View spec: view-spec.md
     - Puffin spec: puffin-spec.md
     - Multi-engine support: multi-engine-support.md
     - How to release: how-to-release.md
     - Terms: terms.md
-  - Concepts:
-    - Catalogs: catalog.md
-  - ASF:
-    - Sponsorship: https://www.apache.org/foundation/sponsorship.html
-    - Events: https://www.apache.org/events/current-event.html
-    - License: https://www.apache.org/licenses/
-    - Security: https://www.apache.org/security/
-    - Sponsors: https://www.apache.org/foundation/thanks.html
+    - ASF: 
+      - Sponsorship: https://www.apache.org/foundation/sponsorship.html
+      - Events: https://www.apache.org/events/current-event.html
+      - License: https://www.apache.org/licenses/
+      - Security: https://www.apache.org/security/
+      - Sponsors: https://www.apache.org/foundation/thanks.html
+  - Concepts: 
```

Comment:
does the re-ordering mean that `ASF` comes before `Concepts`?

Reply:
No, I am nesting ASF under the projects header rather than ASF having their own header.

The main issue wanting the change is how mkdocs renders headers. The previous site we had a hovering dropdown, we're avoiding that now for [accessibility concerns](https://www.w3.org/WAI/tutorials/menus/flyout/) and so clicking on the header requires loading a local page where the sub pages are rendered on the left.

Unfortunately since all the ASF menu items are links, it just brings you to the first sponsorship link rather than show you all the options (you can try this on the current site by clicking the header). To remedy this I have nested these links in the nav menu under project as shown in the second picture above.

This will effectively just remove the ASF from the top nav items which honestly is too crowded as Fokko mentions.